hydra-node/test/Hydra/Chain/Direct/Contract/Mutation.hs

```diff
@@ -728,3 +746,19 @@ replaceHeadId headId = \case
       , Head.headId = headId
       }
   otherState -> otherState
+
+-- | Generates value such that:
+-- - alters between policy id we use in test fixtures with a random one.
```

Comment:
I think that would be a reason to not keep this function "in the framework module". It's specific to our tests / fixtures.

Reply:
I added a dedicated module for contract generators.